### ISSUE TITLE
Variant 0 of Grizzly bear sleeps with eyes open

### ISSUE THREAD

**Issue by CrazyDinoLover** [0SoggyMustache0/ZAWA#157]
Umm... as the title sugests: Variant : 0 of Grizzly bear sleeps with eyes open

**To Reproduce**
Steps to reproduce the behavior. (What were you doing when this happened) :
1. have a Variant: 0 Grizzly Bear
2. set time to night
3. Watch'em sleep

I thought they were gonna close their eyes

(I don't know how to add a screenshot)

**Please complete the following information):**
 - Mod Version Number : 2.1
 - Bookworm Version Number : 2.5.2.1
 - Forge Version Number : 32.0... wait 70? (it says 32.0.70)
 - Are other mods installed? : yes
 - Is OptiFine installed? : yes
 - Is this bug present in single player or on a server or both? :single player

**Comment by CrazyDinoLover** [comment]
is this better?

**Comment by JoelZBub** [comment]
yes. That's better. There was a similar bug on an earlier version. This can go on the to-do list now.
Thanks for the report.

**Comment by CrazyDinoLover** [comment]
> yes. That's better. There was a similar bug on an earlier version. This can go on the to-do list now.
> Thanks for the report
No problem
